packages/back-end/src/init/index.ts

```diff
@@ -9,7 +8,6 @@ export async function init() {
     initPromise = (async () => {
       await mongoInit();
       await queueInit();
-      await initializeLicenseForOrg();
```

Comment:
This is required for the server to function properly, right?  In a horizontally scaled prod environment, are we 100% sure the agenda job will always run immediately at startup?

I don't want this getting blocked by some other jobs for 10 minutes and the instance basically being downgraded to free during that time.

Reply:
The only time this does anything is if the license is stored in env var and it is their very first time loading the server so that there is nothing in the mongo cache.  However there will be no other jobs the first time the server starts up so the job will run right away.  Even if it doesn't they won't be downgraded to free, they would just wait for a single license server call to return.